### DISCUSSION TITLE
Consider turning global/import.ml into a header

### DISCUSSION THREAD

**Discussion by cuihtlauac** [ocaml/ocaml.org#1261] [Ideas]
To reduce the mental load when reading the code, highlighting “something” which looks like `Stdlib` but actually isn't is useful. The module `Import` could be phased out in three steps:

- [ ] Using actual external dependencies instead of blocks of copied code, for instance, get `String.cut` from the package where it's defined and add it as a dependency
- [ ] Move stuff which does not extend actual `Stdlib` modules elsewhere, maybe with a renaming (e.g. `Acc_biggest`)
- [ ] Turn the rest (it should only be oneliners) into a copied header in all the files where it makes sense. That way, understanding remains local, but algorithms which are generic are highlighted as such.

**Comment by sabine** [comment]
Indeed, consolidating all the strewn-around `Stdlib` extensions into one module was a good first step, now to assimilate and rearrange so we don't need to do much `Stdlib` extension at all is great!
